modules/repository-base/base/Makefile

```diff
@@ -65,6 +65,8 @@ GITEPOCH := $(shell git show -s --format=%ct HEAD)
 # Global variables and dirs      #
 ##################################
 
+ARTIFACTS ?= _artifacts
```

Comment:
Add a comment explaining what this variable is used for.
Include the link to that Prow documentation.

`ARTIFACTS` is not used anywhere in *this* repo, so where is it used / how should it be used?
* https://github.com/search?q=repo%3Acert-manager%2Fmakefile-modules%20ARTIFACTS&type=code

Also consider changing the default to `_bin/artifacts` so that all the generated files are in one place and so that I don't have to add another line to the `.gitignore` file of each project.

Reply:
I added a comment explaining what this variable is used for.